README.md

```diff
@@ -141,6 +141,12 @@ sudo systemctl restart pve-cluster.service pvedaemon.service pvestatd.service pv
 
 - Verify Network Connectivity: Ensure that the Proxmox VE nodes can reach the Pure Storage array over the network. Check for firewall rules or network issues that might be blocking communication.
 - Review Logs: Check the Proxmox VE logs for any error messages related to storage or the plugin. Logs are typically found in /var/log/pve.
+  These commands are helpful for troubleshooting:
+  ```bash
+  journalctl -u multipath #diagnose issues with the multipath service
```

Comment:
`multipath -ll -v3` adds more info than journalctl (also there is syntax error -> multipathD)